docs/INSTALLATION.md

```diff
@@ -64,6 +64,35 @@ of the Microsoft Visual C++ compiler.
 
 You should be able to use other integrated IDE's such as JetBrains' CLion in a similar manner.
 
+### Installing Boost on Windows
+
+The `meetup` and `gigasecond` exercises depend on the [Boost](https://www.boost.org/) libraries. The following steps
+describe how to set it up.
+
+The steps assume you install Boost under `C:\Program Files\Boost`, writing here requires Administrator privileges. You
+can install in a different location if you prefer. The steps are for Boost 1.86, but any recent version should work. The
+minimum version required by the exercises is 1.58. If you use a different installation location or version than the ones
+used here, make sure to update the paths accordingly. The steps here are based on the
+[installation guide](https://www.boost.org/doc/libs/1_85_0/more/getting_started/windows.html) at
+[Boost.org](https://www.boost.org/). The steps further assume you have C++ development using Visual Studio configured.
+
+1. [Download Boost](https://www.boost.org/users/download/), downloading the [7-Zip](https://www.7-zip.org/) (.7z)
+archive is [recommended](https://www.boost.org/doc/libs/1_86_0/more/getting_started/windows.html#zip).
+1. Start 7-Zip as Administrator and extract the archive to `C:\Program Files\Boost`.
+1. You should now have a folder `C:\Program Files\Boost\boost_1_86_0` that contains several files and folders.
+1. Add `C:\Program Files\Boost\boost_1_86_0` to the `INCLUDE` environment variable.
```

Comment:
I'm surprised that adding the Boost path to `INCLUDE` makes CMake find it. I'm fairly certain that's not the standard way of doing it. Instead, passing it to CMake via `BOOST_ROOT` I would find more natural.

It looks like with recent CMake releases, there's another complication in that they want to remove (or at least deprecate) the `FindBoost` . But it's probably fine to ignore that for now.

Reply:
I no longer suggest to add anything to `INCLUDE` and use `BOOST_ROOT` instead.